R/pr.R

```diff
@@ -43,13 +43,22 @@ pr_init <- function(branch) {
 #' @export
 #' @rdname pr_init
 #' @param number Number of PR to fetch.
-pr_fetch <- function(number) {
+#' @param owner Name of the owner of the repository that is the target of the
+#'   pull request. Default of `NULL` tries to deduce that from `origin` remote.
+#'
+#' @examples
+#' \dontrun{
+#' ## scenario: current project is a local copy of fork of a repo owned by
+#' ## 'tidyverse', not you
```

Comment:
I've needed this 3 times in the past 2 days, reviewing PRs in readr and dplyr. In both cases, I still have a personal fork.

Reply:
Ah, that makes sense.

I guess we're also going to need a `fork_sitrep()` to tell you if you're in a fork, and if you have write access etc.

Reply:
I haven't done so in these cases, but I think I would be allowed to push into the PRs (internal or external). Because I have write access to the main repo.

Reply:
Yeah, and ideally you don't want to create a PR against your fork, but the original repo. This is going to require some testing 😞